.github/workflows/audit.yaml

```diff
@@ -74,6 +80,12 @@ jobs:
                     "no_of_vulnerabilities"
                   )]
                 ))
+                deps_with_vulnerabilities = subset(renv_lock_scan, no_of_vulnerabilities > 0)
+                if(nrow(deps_with_vulnerabilities) > 0) {
+                  print("Vulnerabilities found in the following dependencies:")
+                  print(deps_with_vulnerabilities["package"])
```

Comment:
I updated the code to use `message()` instead of `print()`.

Additionally, I added `Sys.sleep(1)` in order to avoid race condition with the order of messages being printed, like [here](https://github.com/insightsengineering/r.pkg.template/actions/runs/5520204485/jobs/10066500970).

Final result visible [here](https://github.com/insightsengineering/r.pkg.template/actions/runs/5520384630/jobs/10066933675).